spec/models/item_spec.rb

```diff
@@ -2,12 +2,16 @@
 
 RSpec.describe Item, type: :model do
 
+  let!(:item) {FactoryGirl.create(:item)}
+  let(:item2) {FactoryGirl.create(:item, price: 3,
+              created_at: Date.today, shelf_life_days: -1)}
+
   describe 'validations' do
-    it "is valid with all fields" do
+	  it "is valid with all fields" do
```

Comment:
On these starter "projects" style stuff like this doesn't really matter, but the SB preference is to use spaces over tabs. I think our linting tool, hound, will complain about this. So, if you want, it might be worth spending a few minutes configuring your editor for this.